### ISSUE TITLE
del with parens parses Load instead of Del

### ISSUE THREAD

**Issue by scopatz** [xonsh/xonsh#1234]
So there is a minor bug here:

```
scopatz@artemis ~/xonsh amal $ x, y = 1, 2
scopatz@artemis ~/xonsh amal $ del (x, y)
xonsh: To log full traceback to a file set: $XONSH_TRACEBACK_LOGFILE = <filename>
Traceback (most recent call last):
  File "/home/scopatz/.local/lib/python3.5/site-packages/xonsh/ptk/shell.py", line 101, in push
    locs=None)
  File "/home/scopatz/.local/lib/python3.5/site-packages/xonsh/execer.py", line 108, in compile
    code = compile(tree, filename, mode)
ValueError: expression must have Del context but has Load instead
```

**Comment by scopatz** [comment]
This is valid Python

**Comment by scopatz** [comment]
Potential fix in #2248
